packages/eslint-plugin/src/index.ts

```diff
@@ -9,15 +9,15 @@ const config = {
             parserOptions: {
                 ecmaVersion: 2021,
             },
-            plugins: ['@tybalt'],
+            plugins: ['@tybalt/eslint-plugin'],
```

Comment:
This is recommended here: https://eslint.org/docs/latest/extend/plugins#configs-in-plugins